ui/aura/mus/window_tree_client.cc

```diff
@@ -758,7 +758,8 @@ void WindowTreeClient::OnWindowMusCreated(WindowMus* window) {
   window->set_server_id(MakeTransportId(client_id_, next_window_id_++));
   RegisterWindowMus(window);
 
-  DCHECK(window_manager_delegate_ || !IsRoot(window));
+  DCHECK(window_manager_delegate_ || !IsRoot(window) ||
+         in_external_window_mode_);
```

Comment:
Maybe I am not understanding the possible ChromeOS build implication. could you elaborate?